Explorer/Assets/DCL/PerformanceAndDiagnostics/Diagnostics/ReportsHandling/ReportsHandlingSettingsDevelopment.asset

```diff
@@ -480,6 +480,10 @@ MonoBehaviour:
       Severity: 1
     - Category: LIVEKIT
       Severity: 2
+    - Category: UNSPECIFIED
```

Comment:
you didn't like them in Development? They generally don't bring that much noise but help in development, since you can easily write `Debug.Logs`. I am often get confused why my logs are not appearing :) But I will disable if you have some concerns. Anyway, remember that it is only for Development environments

Reply:
For me, there is no problem at all, but I thought they should not be enabled in the repository (I don't remember who said that).